x-pack/legacy/plugins/snapshot_restore/public/app/services/http/repository_requests.ts

```diff
@@ -30,14 +31,17 @@ export const useLoadRepository = (name: Repository['name']) => {
   });
 };
 
-export const verifyRepository = (name: Repository['name']) => {
-  return sendRequest({
+export const verifyRepository = async (name: Repository['name']) => {
+  const result = await sendRequest({
     path: httpService.addBasePath(
       `${API_BASE_PATH}repositories/${encodeURIComponent(name)}/verify`
     ),
     method: 'get',
-    uimActionType: UIM_REPOSITORY_DETAIL_PANEL_VERIFY,
   });
+
+  const { trackUiMetric } = uiMetricService;
```

Comment:
We can reduce this boilerplate a bit by exporting `trackUiMetric` directly from the `ui_metric` module.